### PR TITLE
Website revamp final

### PR DESCRIPTION
**Pull request by MaanavD**
### Description
Large revamp of the website to use modern technologies. Based on Svelte to be extremely easy to use.

- New Pages, animations, homepage, etc.
- New copy meant for better explanation of ORT

### Motivation and Context
Website in its present form has been around for a long time and presents some challenges with development, this large change is meant to address them. 

### WIP

- Installation table, presently directly links to docs.
- Update docs w/ live
- Weird bug where first page PT/Windows/HF images aren't loading. Addressed soon
- README.md needs work

### Deployment
The website is presently deployed at: https://maanavd.github.io/onnxruntime/

### CODE REVIEW

**Comment by MaanavD** [comment]
Not meant to be merged yet, but will provide as a forum for open input until it's time to merge.

**Comment by natke** [comment]
@MaanavD You can make the PR a draft if you want to ensure it does not get merged.

A couple of notes:
- Need to do a sweep of the current website content to make sure this version contains all of the recent changes (e.g. I notice that the blogs are not up to date). I suggest looking through the GitHub commits that have been made since you created this fork
- Not sure what you mean by docs live? Is it as simple as merging gh-pages?
- Before we merge, can we do a site map comparison to ensure we add redirects for anything significant that has changed?

**Comment by faxu** [comment]
Thanks for getting this major rehaul together! I left a bunch of comments and text revision suggestions. Also, for the main logo in the corner, I prefer we use the shaded logo for branding consistency. Also, "ONNX Runtime" should have a space"

File: src/routes/components/hero.svelte
Comment (MaanavD) [reply src/routes/components/hero.svelte:113]: Made it heavier instead of changing color, will ask for second review before finalizing.

File: src/routes/components/performance.svelte
Comment (MaanavD) [reply src/routes/components/performance.svelte:11]: Happy to link to Olive if you have good wording for it. would love to see if Emma has thoughts

File: src/routes/generative-ai/+page.svelte
Comment (MaanavD) [reply src/routes/generative-ai/+page.svelte:28]: Fixed to be more padded, it's slightly weird regardless though - would love feedback on second review

File: src/routes/generative-ai/+page.svelte
Comment (MaanavD) [reply src/routes/generative-ai/+page.svelte:61]: Images taken from blog post: https://medium.com/microsoftazure/accelerating-stable-diffusion-inference-with-onnx-runtime-203bd7728540. Requires fixing on that end :(

File: src/routes/components/code-blocks.svelte
Comment (prasanthpul) [inline src/routes/components/code-blocks.svelte:20]: the C++ block is much longer than the others and causes extra blank space on the page:
* Change "Load and preprocess the input image to \n// inputTensor, inputNames, and outputNames" to Load and preprocess the input image to inputTensor"
* Change "session.Run(Ort::RunOptions{nullptr}, \ninputNames.data(), \n&inputTensor, \ninputNames.size(), \noutputNames.data(), \noutputNames.size());" to "session.Run(Ort::RunOptions{nullptr}, inputNames.data(), &inputTensor, inputNames.size(), outputNames.data(), outputNames.size());"

File: src/routes/components/videos.svelte
Comment (prasanthpul) [inline src/routes/components/videos.svelte:46]: since 1.14 was many months ago, let's see if we can pick a different video

File: src/routes/inference/+page.svelte
Comment (natke) [inline src/routes/inference/+page.svelte:105]: ONNX Runtime mobile runs models on mobile devices using the same API used for cloud-based inferencing. Developers can use their mobile language and development environment of choice to add AI to Android, iOS, react-native, MAUI/Xamarin applications in Swift, Objective-C, Java, Kotlin, JavaScript, C, and C++.

File: src/routes/components/customers.svelte
Comment (MaanavD) [reply src/routes/components/customers.svelte:68]: Works on live + @https://rad-jelly-4d17cb.netlify.app/, it's an artefact of wanting it to work on github deployments + the actual site. Won't impact use or development, just review.

File: src/routes/components/training-and-inference.svelte
Comment (MaanavD) [reply src/routes/components/training-and-inference.svelte:10]: Inference is above training at https://rad-jelly-4d17cb.netlify.app/ - the linked gh site has to be manually updated every commit, which is rough stuff :(

File: src/routes/inference/+page.svelte
Comment (MaanavD) [reply src/routes/inference/+page.svelte:9]: Fixed.

File: src/routes/inference/+page.svelte
Comment (MaanavD) [reply src/routes/inference/+page.svelte:145]: This is true, but won't exist at any (almost any) real device viewports - a couple pixels larger looks good, a couple smaller changes the sizing. If you think we should swap to a 2x2 grid earlier though, I'm open to doing that!

**Review by prasanthpul — DISMISSED** [review]

File: README.md
Comment (natke) [inline README.md:1]: Do we need this file?

Comment (MaanavD) [reply README.md:1]: The readme will eventually include instructions to make edits to the website!

**Review by prasanthpul — APPROVED** [review]

**Pull request merged**
